contracts/LevCompFactory.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./Strategy.sol";
+
+contract LevCompFactory {
+    address public immutable original;
+
+    event Cloned(address indexed clone);
+    event Deployed(address indexed original);
+
+    constructor(
+        address _vault,
+        address _cToken
+    ) public {
+        Strategy _original = new Strategy(_vault, _cToken);
+        emit Deployed(address(_original));
+
+        original = address(_original);
+    }
+
+    function name() external pure returns (string memory) {
```

Comment:
if we have several factories, adding the apiVersion would help too before cloning?

Reply:
good point, adding it

Reply:
see it was statically added. I will concat the original's version here